prusti-common/src/config.rs

```diff
@@ -101,6 +102,8 @@ lazy_static! {
         settings.set_default("print_collected_verification_items", false).unwrap();
         settings.set_default("hide_uuids", false).unwrap();
         settings.set_default("counterexample", false).unwrap();
+        settings.set_default("print_hash", false).unwrap();
+        settings.set_default("disable_cache", false).unwrap();
```

Comment:
`enable_cache` might be more immediate (one less negation = one less neuron required)